src/module.ml

```diff
@@ -465,21 +425,18 @@ let encode
     ; field_o "kind" Kind.encode kind
     ; field_b "impl" has_impl
     ; field_b "intf" (has_intf t)
-    ; field_i "obj_dir" Obj_dir.encode obj_dir
```

Comment:
I was slightly surprised that we deleted a field here, though I can't see anything wrong with it given that `dune-package` files are not meat to be hand-written.

Reply:
Yeah, but it makes editing the files by hand annoying.